src/Dialect/ONNX/ONNXOps/Math/ElementwiseUnary.cpp

```diff
@@ -280,6 +280,32 @@ LogicalResult ONNXInstanceNormalizationOp::inferShapes(
   return inferShapeForUnaryOps(this->getOperation());
 }
 
+//===----------------------------------------------------------------------===//
+// IsInf
+//===----------------------------------------------------------------------===//
+LogicalResult ONNXIsInfOp::verify() {
+  ONNXIsInfOpAdaptor operandAdaptor(*this);
+  if (!hasShapeAndRank(operandAdaptor.getX()))
+    return success(); // Won't be able to do any checking at this stage.
+
+  int64_t detectPosAttribute = getDetectPositive();
+  int64_t detectNegAttribute = getDetectNegative();
+  
+  // One of the values for detectPosAttribute and detectNegAttribute must be 1. 
+  // If not, then this will result in an error. 
+  if (detectPosAttribute == 0 && detectNegAttribute == 0)
+    return emitOpError("This variation is currently unsupported. One or both of the attributes must be a value of 1 to ensure mapping to infinity.");
```

Comment:
@AlexandreEichenberger As I mentioned to @tungld `IsInf` only supports 3 cases. The fourth scenario which is `detectPosAttribute == 0 && detectNegAttribute == 0` is not mentioned in the documentation. So I added code to the verifier to basically say we do not support that option especially because we have no backend test that even covers it. Having `detectPosAttribute == 0 && detectNegAttribute == 0` would defeat the purpose of the `IsInf` test because everything would be false (nothing will ever be infinity). Are you okay with me adding this verifier here or do you have another suggestion?